components/src/core/components/Input/Time/TimePicker.vue

```diff
@@ -0,0 +1,157 @@
+<template>
+  <div role="alert" class="oxd-time-picker">
+    <div class="oxd-time-hour-input">
+      <oxd-icon-button
+        name="chevron-up"
+        class="oxd-time-hour-input-up"
+        role="none"
+        @click="increment(1, 'hour')"
+        :withContainer="false"
+      />
+      <oxd-input
+        :value="hour"
+        @change="onChange($event, 'hour')"
+        class="oxd-time-hour-input-text"
+      />
+      <oxd-icon-button
+        name="chevron-down"
+        class="oxd-time-hour-input-down"
+        role="none"
+        @click="decrement(1, 'hour')"
+        :withContainer="false"
+      />
+    </div>
+    <div class="oxd-time-seperator">
+      <span class="oxd-time-seperator-icon">:</span>
+    </div>
+    <div class="oxd-time-minute-input">
+      <oxd-icon-button
+        name="chevron-up"
+        class="oxd-time-minute-input-up"
+        role="none"
+        @click="increment(step, 'minute')"
+        :withContainer="false"
+      />
+      <oxd-input
+        :value="minute"
+        @change="onChange($event, 'minute')"
+        class="oxd-time-minute-input-text"
+      />
+      <oxd-icon-button
+        name="chevron-down"
+        class="oxd-time-minute-input-down"
+        role="none"
+        @click="decrement(step, 'minute')"
+        :withContainer="false"
+      />
+    </div>
+    <div class="oxd-time-period-input">
+      <div class="oxd-time-period-label">
+        <input name="am" v-model="period" type="radio" value="AM" />
+        <label for="am">AM</label>
```

Comment:
How about localization

Reply:
Maybe all components should include trans propose .. but I guess it's out of scope